config/config.go

```diff
@@ -35,10 +35,12 @@ type (
 		}
 
 		Agent struct {
-			Host        string
-			Token       string
-			Image       string `default:"drone/agent:0.8"`
-			Concurrency int    `default:"2"`
+			Host             string
+			Token            string
+			Image            string `default:"drone/agent:0.8"`
+			Concurrency      int    `default:"2"`
+			KeepaliveTime    string `default:"6m"`
```

Comment:
Both types should be time.Duration

Reply:
Done. I was having a bit of trial and error and some missunderstandings.
I'll try this PR (`f252e2a`) on our environment now

Reply:
It doesn't build in our environment. I think I had this before, a `git commit --amend` and force push actually yields the previous commit beeing actually built/tested.

I had tried time.Duration before and got the same error infact hence I changed it to strings.
I believe time.Duration if casted to a string get's printed out as an int in nanoseconds (though I surely can fix that in the print statement and print it as say 300s (in seconds basically)).

I'll carry on tomorrow I guess, thanks for the quick feedback through! Appreciated.

The error is as follows: 

```
=== RUN   TestLoad
--- FAIL: TestLoad (0.00s)
	load_test.go:138: json: cannot unmarshal string into Go struct field .KeepaliveTime of type time.Duration
FAIL
```

Reply:
I changed it to nanoseconds in the load_test. That should build successfully